scripts/manage/landing_page_base.md

```diff
@@ -114,6 +114,10 @@ accounts) must be pushed and applied to your deployment to take effect:
 
 ### Add Spinnaker account for GKE
 
+This script grants the Spinnaker deployment's Google Service Account the required
+[GKE IAM roles](https://cloud.google.com/kubernetes-engine/docs/how-to/iam) in the
```

Comment:
"GKE IAM roles"
Do we need "GKE" here?

Reply:
Here's a possible rewrite of that sentence, assuming "Spinnaker deployment" --> "Spinnaker instance" and we don't need "GKE":

"This script grants the required IAM roles to the Spinnaker instance's service account, in the GCP project containing the referenced cluster."

Reply:
I like it. Will change it in all 3 locations.

Reply:
Done.